backend/app/routes/auth.py

```
@@ -0,0 +1,356 @@
+    refresh_in: RefreshRequest,
+    session: Session = Depends(get_session)
+):
+    credentials_exception = HTTPException(
+        status_code=status.HTTP_401_UNAUTHORIZED,
+        detail="Could not validate credentials",
+        headers={"WWW-Authenticate": "Bearer"},
+    )
+
+    # Verify token signature
+    try:
+        payload = jwt.decode(refresh_in.refresh_token, settings.JWT_SECRET_KEY, algorithms=[settings.JWT_ALGORITHM])
```

Ruff (0.14.10)

Undefined name `jwt`

(F821)